pyproject.toml

```
@@ -28,7 +28,7 @@ requires-python = "~=3.7"
 dependencies = [
     "click>=7.1,<9",
     "pyyaml",
-    "sphinx>=3,<5",
+    "sphinx>=3,<6",
```

```suggestion
    "sphinx>=4,<6",
```

@coretl you are correct re: style guide. I think this should be `>=4,<6`

---

@AakashGfude feel free to merge after your review. We can update the lower bound as part of the release.

---

This has been resolved